test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/NpgsqlCommandTests.cs

```diff
@@ -0,0 +1,40 @@
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AdoNet
+{
+    public class NpgsqlCommandTests : TestHelper
```

Comment:
Also, this file should replace `NpgSqlTests.cs` right? I expected that file to be deleted.

Reply:
Thanks. I missed this when I moved the file. Deleted in 388f0745d0286f815096c8440c0e3502753e59d4.